nservicebus/azure-servicebus/topologies/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Azure Service Bus Topologies
+summary: Azure Service Bus Topologies
+tags:
+- Azure
+- Cloud
+redirects:
+related:
+ - samples/azure/azure-service-bus
+reviewed: 2016-04-12
+---
+
+Messaging topology is an arrangement of the various messaging entities.
+Azure Service Bus transport operates on a topology created on the broker. Messaging entities participating in topology are queues, topics, subscriptions, and rules. Combined in a specific way they form a topology. 
+
+Topology handles message exchange between endpoints by creating and configuring Azure Service Bus entities.  
+
+## Version 6 and below
+
+Version 6 and below of the transport has always supported a single default topology out-of-the-box. It was an implicit implementation of what was defined as `EndpointOrientedTopology` in version 7 and up.
+
+## Version 7 and up
+
+With version 7 of the transport, topology concept was introduced. There are two topologies:
+
+1. `EndpointOrientedTopology`
+1. `ForwardingTopology`
+
+`EndpointOrientedTopology` is the default topology. It's fully backwards compatible with version 6 and below of the transport.`ForwardingTopology` was introduced to take advantage of the broker nature of Azure Service Bus and leverage the native capabilities. This makes `ForwardingTopology` a much better option for new projects. 
+
+Both topologies create input queue per endpoint and implement pub/sub mechanism. Significant difference between the two is how pub/sub mechanism is implemented.
+
```

Comment:
The `EndpointOrientedTopology` is backward compatible with Azure Service Bus transport Version 6 and lower, meaning that endpoints using Version 5 can communicate with endpoints using Version 6 (is that right?)/ meaning that endpoints using Version 5 can be upgraded to Version 6 without changes in behavior (?). It is the recommended option when upgrading to Version 6 or higher.

The `ForwardingTopology` was introduced to take advantage of the broker nature of Azure Service Bus and to leverage its native capabilities. It is the recommended option for new projects.

Both topologies create a single input queue per endpoint and implement Publish-Subscribe mechanism. However, there is a significant difference in the way the mechanism is implemented in each of them.